bennettbot/job_configs.py

```diff
@@ -324,44 +323,53 @@
                 "report_format": "blocks",
             },
             "show_failed": {
+                "run_args_template": "python jobs.py show --target {target} --skip-successful",
+                "report_stdout": True,
+                "report_format": "blocks",
+            },
+            "show_failed_all": {
                 "run_args_template": "python jobs.py show --target all --skip-successful",
                 "report_stdout": True,
                 "report_format": "blocks",
             },
         },
         "slack": [
-            {
-                "command": "check-links",
-                "help": "Summarise GitHub Actions workflow runs for link-checking workflows in website repos.",
-                "action": "schedule_job",
-                "job_type": "check_links",
-            },
             {
                 "command": "key",
                 "help": "Show the emoji key being used in the workflow summaries.",
                 "action": "schedule_job",
                 "job_type": "display_emoji_key",
             },
             {
-                "command": "show all",
+                "command": "usage",
+                "help": "Show help message for the `show [target]` and `show-failed [target]` commands.",
+                "action": "schedule_job",
+                "job_type": "display_usage",
+            },
+            {
+                "command": "show",
                 "help": "Summarise GitHub Actions workflow runs for repos in all organisations.",
                 "action": "schedule_job",
                 "job_type": "show_all",
             },
             {
                 "command": "show-failed",
-                "help": "Summarise GitHub Actions workflow runs for repos in all organisations, skipping repos whose runs are all successful.",
+                "help": "Summarise GitHub Actions workflow runs for repos in all organisations, but skips summarising repos whose runs are all successful.",
```

Comment:
Maybe "Summarise unsuccessful GitHub Actions workflow runs for repos in all organisations"?